lib/jira_wrapper.rb

```diff
@@ -98,6 +98,17 @@ def self.get_password(app_name, username)
       response.to_s.empty? ? nil : response
     end
 
+    def create_version!(release_name, project:, released: "true")
+      begin
+        version = @jira_client.Version.build
+        version.save({ "project" => project, "name" => release_name, "released" => released })
+        version.attrs["id"]
```

Comment:
if we stick with this method in jira-glue.. I'd be tempted to return the version object.. and that way consumers can decide what to do w/ the version from here